client/app/email-domain/email-domain.service.js

```diff
@@ -1091,13 +1091,27 @@ angular.module("services").service(
          * @param {string} destinationEmailAddress
          */
         checkMigrate (domain, accountName, destinationServiceName, destinationEmailAddress) {
-            return this.OvhHttp.get(`/email/domain/${domain}/account/${accountName}/migrate/${destinationServiceName}/destinationEmailAddress/${destinationEmailAddress}/checkMigrate`, {
-                rootPath: "apiv6"
-            });
+            const defer = this.$q.defer();
```

Comment:
We should avoid to do this. The API have to return an error state :)

Can we discuss about that when you're available ? :)